cdc/log.hh

```diff
@@ -43,6 +43,7 @@
 #include "tracing/trace_state.hh"
 #include "cdc_options.hh"
 #include "utils/UUID.hh"
+#include "stats.hh"
```

Comment:
Why is this include needed here if stats are only used in log.cc?

Reply:
Removed